common/helpers/g2p.py

```diff
@@ -0,0 +1,100 @@
+__all__ = ["G2PBasedOovAugmenter"]
+
+import os
+from typing import Optional
+
+from sisyphus import tk
+Path = tk.setup_path(__package__)
+
+from i6_core.corpus.stats import ExtractOovWordsFromCorpusJob
+from i6_core.g2p.apply import ApplyG2PModelJob
+from i6_core.g2p.convert import BlissLexiconToG2PLexiconJob, G2POutputToBlissLexiconJob
+from i6_core.g2p.train import TrainG2PModelJob
+
+
+class G2PBasedOovAugmenter():
+    def __init__(self,
+                 original_bliss_lexicon:str,
```

Comment:
there should be space after `:` because this violates PEP8. Black formatting does not fix this?